source/adapters/level_zero/device.cpp

```diff
@@ -861,6 +861,8 @@ UR_APIEXPORT ur_result_t UR_APICALL urDeviceGetInfo(
     return ReturnValue(ur_bool_t{false});
   case UR_DEVICE_INFO_IMAGE_SRGB:
     return ReturnValue(ur_bool_t{false});
+  case UR_DEVICE_INFO_KERNEL_SET_SPECIALIZATION_CONSTANTS:
+    return ReturnValue(false);
```

Comment:
Shouldn't this be ur_bool_t{false}?

Reply:
In the time between creating this MR and now, it looks as if this change has been added via 646cc9ce7 , so this MR no longer touches level zero. There's a lot of device infos that return `bool`s rather than `ur_bool_t` here, so it might be worth having a separate change that updates all of them at once.